src/system/RoutesModule/Resources/config/customServices.yml

```diff
@@ -20,17 +23,17 @@ services:
     zikula_routes_module.routing_loader:
         class: Zikula\RoutesModule\Routing\RouteLoader
         arguments:
-            - "@doctrine.orm.entity_manager"
-            - "@service_container"
+            - "@doctrine.orm.default_entity_manager"
+            - "@translator"
```

Comment:
I _think_ the zikula translator is just an extension of symfony's. So it could still be used.